charts/internal/calico/templates/multus/daemonset-multus-cni.yaml

```diff
@@ -0,0 +1,136 @@
+{{- if .Values.config.multus.enabled }}
+---
+# This manifest installs Multus CNI plugins and network config on
+# each node in a Kubernetes cluster.
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus
+  namespace: kube-system
+  labels:
+    k8s-app: multus
+    gardener.cloud/role: system-component
+    node.gardener.cloud/critical-component: "true"
+  {{- if not .Values.autoscaling.staticRequests }}
+  annotations:
+    resources.gardener.cloud/preserve-resources: 'true'
+  {{- end }}
+spec:
+  revisionHistoryLimit: 5
+  selector:
+    matchLabels:
+      k8s-app: multus
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 2
+  template:
+    metadata:
+      labels:
+        node.gardener.cloud/critical-component: "true"
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-apiserver: allowed
+        networking.gardener.cloud/to-dns: allowed
+        k8s-app: multus
+        gardener.cloud/role: system-component
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+      hostNetwork: true
+      hostPID: true
+      tolerations:
+        # Make sure multus gets scheduled on all nodes.
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+      serviceAccountName: multus
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+      # Minimize downtime during a rolling upgrade or deletion; tell Kubernetes to do a "force
+      # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
+      terminationGracePeriodSeconds: 0
+      priorityClassName: system-node-critical
+      initContainers:
+      # This container installs the multus CNI binaries
+      # and CNI network config file on each node.
+      - name: install-multus-cni
+        image: {{ index .Values.images "multus-cni" }}
+        command: ["/install_multus"]
+        args:
+          - "--type"
+          - "thin"
+        resources:
+          requests:
+            cpu: 10m
+            memory: 15Mi
+        volumeMounts:
+          - name: cnibin
+            mountPath: /host/opt/cni/bin
+            mountPropagation: Bidirectional
+        securityContext:
+          privileged: true
+      {{ if .Values.config.multus.installCNIPlugins }}
+      # This container installs the CNI binaries on each node.
+      - name: install-cni-plugins
```

Comment:
What does this container install? I thought that `multi-install-cni` container would install multus. What is copied here?